OracleDatabase/SingleInstance/dockerfiles/19.3.0/Dockerfile

```diff
@@ -22,9 +22,17 @@
 # ---------------
 FROM oraclelinux:7-slim as base
 
-# Maintainer
-# ----------
-MAINTAINER Gerald Venzl <gerald.venzl@oracle.com>
+# Labels
+# ------
+LABEL "provider"="Oracle"                                               \
```

Comment:
Is maintainer part of the Label?

Reply:
No, it's not. @gvenzl and I discussed this and we agreed that having individual names wasn't ideal for long-term sustainability. Though, you're free to add a maintainer label to your images if you want.

Reply:
Do you think this is a big issue, @mriccell?

My rationale was the following:

I didn't mind the individual names at first, just to provide a way for the maintainer(s) to get some external prestige and because they were already there in the old `MAINTAINER` instruction. Avi then had a good point that individual names may cause issues with continuity, a person leaving Oracle, etc. So we then thought of having team names, e.g.`maintainer="Oracle Database Team`. However, I felt that adding "Oracle Database Team as maintainer provides little value add to `provider="Oracle"`, after all, who else would or should be providing these images if not the team of the product itself? And frankly, why would customers care what team inside Oracle provides these images and why should they? Internal org charts shouldn't be reflected in such public images IMO. So this is when we then opted for "well, does maintainer even make any more sense, probably not."

However, if you have good reason to believe that maintainer is important, please let us know!